gui/wxpython/gui_core/prompt.py

```diff
@@ -300,11 +304,40 @@ def OnItemSelected(self, event):
             except IOError:
                 self.cmdDesc = None
 
+    def IsHintShown(self):
+        return self._hint_shown
+
+    def SetHintShown(self, shown):
+        self._hint_shown = shown
```

Comment:
Just some minor things. You don't need these methods, they are not supposed to be called from outside, just directly set the attribute. Also methods ShowHint/HideHint could be _showHint/_hideHint since they are used only within the class.